src/libraries/System.Text.Json/gen/JsonSourceGenerator.Parser.cs

```diff
@@ -733,6 +735,12 @@ private TypeGenerationSpec GetOrAddTypeGenerationSpec(Type type, JsonSourceGener
                         numberHandling = (JsonNumberHandling)ctorArgs[0].Value!;
                         continue;
                     }
+                    else if (attributeTypeFullName == JsonUnmappedMemberHandlingAttributeFullName)
+                    {
+                        IList<CustomAttributeTypedArgument> ctorArgs = attributeData.ConstructorArguments;
+                        unmappedMemberHandling = (JsonUnmappedMemberHandling)ctorArgs[0].Value!;
+                        continue;
```

Comment:
This logic is identical to the above. Should it be abstracted to a generic method?

Reply:
Maybe, but this is only generalizable to attributes containing a single constructor parameter (and not other constructor arities or property setters). We might consider it if it becomes pervasives, but only two instances probably doesn't justify extracting to a helper (it probably wouldn't have any substantial impact on LOC and would aversely impact readability).